db/seeds.rb

```diff
@@ -22,3 +28,32 @@
 stache_4.categories << cat_4
 stache_5.categories << cat_1
 stache_6.categories << cat_2
+
+user = User.create(username: "beth", password: "123")
+stache_7 = Stache.create(name: "Mustachio",
+                         description: "things",
+                         price: 50.00,
+                         retired: true,
+                         image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
+stache_8 = Stache.create(name: "The Lenny",
+                        description: "A true mustache.",
+                        price: 5000.00,
+                        retired: true,
+                        image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
```

Comment:
Line is too long. [99/80]<br>Align the elements of a hash literal if they span more than one line.